angr/analyses/cfg/cfg_fast.py

```diff
@@ -1675,6 +1675,14 @@ def _func_addrs_from_prologues(self):
         for ins_regex in self.project.arch.function_prologs:
             r = re.compile(ins_regex)
             regexes.append(r)
+        # FIXME: HACK: Oh my god i'm sorry
```

Comment:
@subwire What is this "FIXME" "HACK" for?

Reply:
Ah, yes, this is referring to the thumb prologs thing, which seems like a disgusting hack.  Now, it turned out in the end that it worked, and I can't think of a better way, so I guess I'll just take that out?
If you have alternatives that are more generic that would be cool.

Reply:
Reworded.